rca/home/models.py

```diff
@@ -1,29 +1,187 @@
+        blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    hero_colour_option = models.CharField(
+        max_length=1,
+        choices=(
+            ("1", "Light text on a dark image"),
+            ("2", "Dark text on a light image"),
+        ),
+    )
+    hero_cta_url = models.URLField(blank=True)
+    hero_cta_text = models.CharField(max_length=125, blank=True)
+    hero_cta_sub_text = models.CharField(max_length=125, blank=True)
 
-    search_fields = BasePage.search_fields + [index.SearchField("strapline")]
+    strapline = models.CharField(max_length=125)
+    strapline_cta_url = models.URLField(blank=True)
+    strapline_cta_text = models.CharField(max_length=125, blank=True)
 
     content_panels = BasePage.content_panels + [
-        FieldPanel("strapline"),
-        SnippetChooserPanel("call_to_action"),
+        MultiFieldPanel(
+            [
+                ImageChooserPanel("hero_image"),
+                FieldPanel("hero_colour_option"),
+                FieldPanel("hero_cta_url"),
+                FieldPanel("hero_cta_text"),
+                FieldPanel("hero_cta_sub_text"),
+            ],
+            heading="Hero",
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("strapline"),
+                FieldPanel("strapline_cta_url"),
+                FieldPanel("strapline_cta_text"),
+            ],
+            heading="Strapline",
+        ),
+        InlinePanel("transformation_blocks", label="Transormation block", max_num=1),
+        InlinePanel("partnerships_block", label="Partnerships", max_num=1),
+        InlinePanel("stats_block", label="Statistics", max_num=1),
     ]
+
+    def clean(self):
+        errors = defaultdict(list)
+        if self.hero_cta_url and not self.hero_cta_text:
+            errors["hero_cta_text"].append(
+                "Please add the text to be displayed as a link"
+            )
+        if self.hero_cta_text and not self.hero_cta_url:
+            errors["hero_cta_url"].append("Please add a URL value")
+        if self.strapline_cta_url and not self.strapline_cta_text:
+            errors["hero_cta_text"].append(
+                "Please add the text to be displayed as a link"
+            )
+        if self.strapline_cta_text and not self.strapline_cta_url:
+            errors["hero_cta_url"].append("Please add a URL value")
+
+        if errors:
+            raise ValidationError(errors)
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context["hero_colour"] = "dark"
+        if self.hero_colour_option == "1":
+            context["hero_colour"] = "light"
+
+        context["transformation_block"] = self.transformation_blocks.select_related(
+            "image"
+        ).first()
+        context["partnerships_block"] = self.partnerships_block.first()
+        context["stats_block"] = self.stats_block.select_related("background_image")
```

Comment:
If you want to pass a queryset, you should reflect it in the name and call is `stats_blocks`. But it seems like you only want the first block anyways, so why not just pass `first()` here?